optuna/samplers/_cmaes.py

```diff
@@ -168,21 +163,24 @@ def objective(trial):
             increased or decreased.
             Please see also ``inc_popsize`` parameter.
 
-            .. note::
-                Added in v2.1.0 as an experimental feature. The interface may change in newer
-                versions without prior notice. See
-                https://github.com/optuna/optuna/releases/tag/v2.1.0.
+            .. warning::
+                Deprecated in v4.4.0. ``restart_strategy`` argument will be removed in the future.
+                The removal of this feature is currently scheduled for v6.0.0,
+                but this schedule is subject to change.
+                See https://github.com/optuna/optuna/releases/tag/v4.4.0.
+                `restart_strategy` will be supported in OptunaHub.
 
         popsize:
-            A population size of CMA-ES. When ``restart_strategy = 'ipop'``
-            or ``restart_strategy = 'bipop'`` is specified,
-            this is used as the initial population size.
+            A population size of CMA-ES.
 
         inc_popsize:
             Multiplier for increasing population size before each restart.
             This argument will be used when ``restart_strategy = 'ipop'``
             or ``restart_strategy = 'bipop'`` is specified.
 
+            .. warning::
+                Deprecated along with ``restart_strategy``.
```

Comment:
Deprecation messages should be written at the same level of text as restart_strategy without simplification.